### DISCUSSION TITLE
Few customers using specific ISP not able to access Firebase

### DISCUSSION THREAD

**Discussion by sbijral** [firebase/firebase-js-sdk#4970] [Q&A] [answered]
![Firebase_Issue](https://user-images.githubusercontent.com/48199030/120275333-c5a66500-c2ce-11eb-8daa-14c36e89989b.png)
As an organisation, we provide the end customer a chat platform using  Firebase RTDB. From 27 May 2021, a lot of customers using a specific ISP in Indonesia(all the customers are from Indonesia) are facing issue while loading the chat. There is no change on the application side.If the customer switch to some other network/ISP, they are able to see the chat and when they go back to the the ISP which had issue, they are able to see the issue again. For some of the customers, the problem has disappeared abruptly without any fix from the application side itself. How can I debug such scenario in Firebase.

**Comment by vkryachko** [comment]
hi @sbijral , does your chat app use Android SDKs?
(from the screenshot I assume that you're experiencing this issue in your web app, not an android app)

**Comment by swastikchugh** [comment]
Hey @vkryachko 
We are not using any Android SDKs in this app. Its in the web app, and for some of the users it wasnt loading on a certain day, but started to load up after 1-2 days, but certain users are still facing this issue. 
I believe that the issue might be similar to [https://github.com/firebase/firebase-ios-sdk/issues/4913](https://github.com/firebase/firebase-ios-sdk/issues/4913), but im not entirely sure.

**Reply by vkryachko** [reply]
I see, I am going to transfer this discussion to http://github.com/firebase/firebase-js-sdk , hopefully web folks will be able to help triage the issue you're having

**Comment by samtstern** [comment, marked as answer]
@sbijral a few things:

- I think you posted this on the wrong repo, so I have transferred it to our web repo `firebase-js-sdk`
- The SDK team is unlikely to be able to help you with an ISP issue. Please contact Firebase Support (https://firebase.google.com/support) or the ISP

**Comment by RaheutRahwana** [comment]
Hi @sbijral 
May I know the ISP? I would like to test my Firebase apps on that too. Thanks!

NB: Salam dari Bandung...

**Reply by sbijral** [reply]
It was happening for Indihome and Telkomsel ISPs
